includes/Generator/Plugins/SchemaOrg.php

```diff
@@ -123,19 +125,35 @@ public function addMetadata() {
 			}
 		}
 
+
 		$meta['image'] = $this->getImageMetadata();
 		$meta['author'] = $this->getAuthorMetadata();
 		$meta['publisher'] = $this->getAuthorMetadata();
 		$meta['potentialAction'] = $this->getSearchActionMetadata();
 
 		$this->outputPage->addHeadItem( 'jsonld-metadata', sprintf( $template, json_encode( $meta ) ) );
 	}
+	
+	/**
+	* Generate proper schema.org type in order to pass validation
+	* 
+	* @return string
+	*/
+	
+	private function getTypeMetadata() {
+		$data = $this->metadata['type'];
+		if (!isset ($this->metadata['type'])) {
+			$data = "article";
+		}
+		return $data;
+	}
 
 	/**
 	 * Generate jsonld metadata from the wiki logo or supplied file name
 	 *
 	 * @return array
 	 */
+	 
 	private function getImageMetadata() {
```

Comment:
I think the method should try to set the sites logo if no image is set.  
See the comment from line +191 to +206 in the ```getAuthorMetadata()``` method. :)